app/views/playlists/_share.html.erb

```diff
@@ -13,7 +13,7 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-<div id="share-button">
+<div id="share-button" style="margin: -1.6em 1em 1em 0;">
```

Comment:
@bkeese I think it would be better if we could move this inline styling to a CSS file.

Reply:
Agreed. Will you fix please?